### PR TITLE
Mark `under13` and `Next` and `Back` buttons in signup for localisation

### PR DESCRIPTION
**Pull request by surfdude29**
This PR marks the `under13` string and the `Next` and `Back` buttons (and their labels) in the signup flow for translation.

### CODE REVIEW

File: src/screens/Signup/index.tsx [hunk 1]
```diff
@@ -170,34 +170,36 @@ export function Signup({onPressBack}: {onPressBack: () => void}) {
...
                     <Button
-                      label="Retry"
+                      label={_(msg`Press to retry`)}
                       variant="solid"
                       color="primary"
                       size="medium"
                       disabled={state.isLoading}
                       onPress={() => refetch()}>
-                      Retry
+                      <Trans>Retry</Trans>
```
Comment (haileyok) [inline src/screens/Signup/index.tsx:190]: Can you please wrap this in `ButtonText` 🙏

File: src/screens/Signup/index.tsx [hunk 2]
```diff
@@ -170,34 +170,36 @@ export function Signup({onPressBack}: {onPressBack: () => void}) {
...
             <View style={[a.flex_row, a.justify_between, a.pb_lg]}>
               <Button
-                label="Back"
+                label={_(msg`Go back to previous step`)}
                 variant="solid"
                 color="secondary"
                 size="medium"
                 onPress={onBackPress}>
-                Back
+                <Trans>Back</Trans>
```
Comment (haileyok) [inline src/screens/Signup/index.tsx:178]: This one too please.

Comment (surfdude29) [reply src/screens/Signup/index.tsx]: Done 👍

**Review by haileyok — APPROVED** [review]
Thank you!

**Comment by surfdude29** [comment]
Thanks Hailey! :)

**Pull request merged**